proxy/src/grpc/sql_query.rs

```diff
@@ -144,13 +141,12 @@ impl<Q: QueryExecutor + 'static, P: PhysicalPlanner> Proxy<Q, P> {
         };
 
         let (tx, rx) = mpsc::channel(STREAM_QUERY_CHANNEL_LEN);
-        let runtime = ctx.runtime.clone();
         let resp_compress_min_length = self.resp_compress_min_length;
         let output = self
             .as_ref()
-            .fetch_sql_query_output(ctx, &schema, &req.sql)
+            .fetch_sql_query_output(ctx, schema, &req.sql, false)
             .await?;
-        runtime.spawn(async move {
+        Handle::current().spawn(async move {
```

Comment:
Current time is the `ctx.runtime`? I guess using the `ctx.runtime` is better for clear.

Reply:
Currently there is no this requirements, and if we want to support, we can add explicit runtime to specific API.

Context is used in all request type, like route/query/write/stream, and only stream API have this requirement, I think put runtime in context is little heavy.

Reply:
That's ok to separate the runtime from the context, but for some cases, switching to a new runtime to execute the task is necessary and this mechanism should be kept.